apps/crunchy-postgres/tests/node_affinity/run_litmus_test.yml

```diff
@@ -0,0 +1,37 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: litmus-node-affinity
+  namespace: litmus 
+spec:
+  template:
+    metadata:
+      name: litmus
+      labels:
+        app: node_affinity
+   
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+
+        env: 
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays
+            value: default
+
+          - name: NAMESPACE
+            value: litmus
+
+          - name: APPLICATION_LABEL
+            value: app=pgset
+
+          - name: PVC_NAME
+            value: crunchy-claim   
```

Comment:
@shashank855 use APP_PVC instead off PVC_NAME

Reply:
sure, I will make the chenges.